code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -751,6 +751,9 @@
 		addtimer(CALLBACK(src, .proc/shootAt, target), 15)
 		return TRUE
 
+/obj/machinery/porta_turret/syndicate/pod/toolbox
+	faction = list(ROLE_TRAITOR)
```

Comment:
this is pretty bad because its just a traitor tester, you should do something like a var for the person who deployed it which is set on the wrench function and then in initialize do faction |= "[REF(master)]" to make the deployer the only one safe

Reply:
and maybe add a way to use the wrench to designate people as being friends?

Reply:
you should also make it so that bashing this with a wrench heals it instead of damages it

and since this is an 11 TC item, maybe you should make it drop its toolbox upon destruction?